sql_metadb/report_queries/administration/user_permission_sets.sql

```diff
@@ -0,0 +1,32 @@
+WITH permissions_users AS (
+    SELECT
+        jsonb_array_elements_text(permissions_users.jsonb -> 'permissions') AS user_permissions,
+        jsonb_extract_path_text(permissions_users.jsonb, 'userId')::uuid AS user_id
+    FROM
+        folio_permissions.permissions_users AS permissions_users
+)
+SELECT
+    permissions.id AS permission_id,
+    jsonb_extract_path_text(permissions.jsonb, 'displayName') AS permission_name,
+    jsonb_extract_path_text(permissions.jsonb, 'description') AS permission_description,
+    users.id AS user_id,
+    jsonb_extract_path_text(users.jsonb, 'personal', 'lastName') AS user_lastname,
+    jsonb_extract_path_text(users.jsonb, 'personal', 'firstName') AS user_firstname,
+    jsonb_extract_path_text(users.jsonb, 'barcode') AS user_barcode,
+    jsonb_extract_path_text(users.jsonb, 'username') AS user_username,
+    CASE WHEN jsonb_extract_path_text(users.jsonb, 'active') = 'true' THEN
```

Comment:
I think my preference here would be to cast the left expression to `boolean`.  I will make the change here.